runtime-modules/content/src/lib.rs

```diff
@@ -189,6 +192,11 @@ decl_storage! {
 
         /// Max nft auction whitelist length
         pub MaxAuctionWhiteListLength get(fn max_auction_whitelist_length) config(): MaxNumber;
+
+        /// Bids for open auctions
+        pub BidByVideoAndMember get(fn bid_by_video_and_member):
```

Comment:
`OpenAuctionBidByVideoAndMember`

Reply:
renaming done in 6f3f47bc3dc229e7af3d3b00f4df7f68e94c4cdd